src/peft/config.py

```diff
@@ -23,6 +23,30 @@
 from .utils import CONFIG_NAME, PeftType, TaskType
 
 
+@dataclass
+class PeftConfigRuntime:
+    """
+    This is the sub-configuration class to store the runtime configurations for the model.
+
+    Args:
+        ephemeral_transfers (`bool`): Whether to use ephemeral transfers for models partially kept in CPU memory.
+    """
+    ephemeral_transfers: bool = field(
```

Comment:
I'm still wondering about the name. Maybe we can find something that is a bit more specific. Just as an example, it could be `optimize_device_memory` or something like this. Of course, we may add more optimizations in the future, so it's hard to predict if that name will fit in the future. But I feel like users will be confused when reading "ephemeral transfers".

Reply:
I agree that the name is probably not good as is. `optimize_device_memory` sounds like it decreases VRAM usage, when in reality it gives a tiny momentary overhead. I'll think of "normal" succinct ways of saying it.

Reply:
I see your point. "optmize" does not necessarily mean "decrease", but I agree that this is what many users might think. LMK if you come up with a better name.

Reply:
this sounds more or less (but I agree not exactly) as some sort of temporary CPU offloading. What about something like `ephemeral_cpu_offload` ?

Reply:
Well, the idea is to keep it in CPU memory all the time and, just as it is needed for some heavy calculation, copy it to GPU, and then remove it when finished. "Temporary CPU offloading" kind of sounds like the opposite of that, i.e. it's kept on GPU and then temporarily put in CPU memory for brief periods.